lib/main.rb

```diff
@@ -1,3 +1,7 @@
 require './manipulate_file'
 
-print ManipulateFile.new("../games.log").get_first_line
+file = ManipulateFile.new("../games.log")
+
+obj = { file.name => { "lines" => file.qty_lines} }
+
+puts file.mount_json(obj)
```

Comment:
Configura seu editor para adicionar EOF (End of file) quando vc salvar um arquivo, pode ser que isso te cause algum problema.